File: tests/Contexts/Rutine/Exercises/domain/ExerciseIdMother.ts
import { ExerciseId } from '../../../../../src/Contexts/Rutine/Exercises/domain/ExerciseId';
import { WordMother } from '../../../Shared/domain/WordMother';

export class ExerciseIdMother {
  static create(value: string): ExerciseId {
    return new ExerciseId(value);
  }

  static random(): ExerciseId {
    return this.create(WordMother.random());
  }
}

File: src/test/java/com/kuro/rutineweb/exercises/entities/ExerciseIdMother.java
package com.kuro.rutineweb.exercises.entities;

import com.kuro.rutineweb.Shared.entities.WordMother;

public class ExerciseIdMother {
    public static ExerciseId create(String value) {
        return new ExerciseId(value);
    }

    public static ExerciseId random() {
        return new ExerciseId(WordMother.random(12));
    }
}
